1.1-schnorr-signatures.ipynb

```diff
@@ -28,19 +29,19 @@
    "source": [
     "## Part 1: Schnorr Signatures\n",
     "\n",
-    "[bip-schnorr](https://github.com/bitcoinops/bips/blob/V0.1/bip-schnorr.mediawiki) defines a signature/verifier scheme, as well as pubkey and signature encodings.\n",
+    "[bip-0340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) defines a signature/verifier scheme, as well as pubkey and signature encodings.\n",
     "\n",
     "The schnorr signature equation is the following:\n",
     "\n",
-    "* `S = R + H(x(R)|P|m) * P`\n",
+    "* `S = R + H(x(R)|x(P)|m) * P`\n",
```

Comment:
ebb8f52e06bdd6e77fac5c0d3694f8a4414f2442
Agree with this, since the equation notes that it takes the x-coord of nonce `R`, it should have consistent notation for pubkey `P`

Just wondering why the notation isn't `$P_x$` (but like, with a subscript)?

Reply:
yeah i agree with you that the subscript notation looks better. Will update 👍

Reply:
Am gonna leave these as is for now cause I see that this format is used in the BIP. Not sure why it is done like that, but i think it makes sense to have the notation the same as the that of the BIP.